### ISSUE TITLE
input type=number, type=range should report a violation for use of aria-valuemin

### ISSUE THREAD

**Issue by carmacleod** [IBMa/equal-access#360]
Steps:
- go to https://w3c.github.io/html-aria/tests/min-test.html
- run Checker

input type=number and type=range should report a violation for use of aria-valuemin

**Comment by MHoov** [comment]
As of June 1
- [x] general rule description (SME & Dev)
- [ ] testcases written (Dev)
- [ ] rule bug (Dev)
- [x] message written (SME)
- [x] help written (SME)

**Comment by shunguoy** [comment]
this should be covered in #721.  close.
